launcher/src/main/java/org/apache/spark/launcher/AbstractAppHandle.java

```diff
@@ -72,11 +74,7 @@ public void stop() {
   @Override
   public synchronized void disconnect() {
     if (connection != null && connection.isOpen()) {
-      try {
-        connection.close();
-      } catch (IOException ioe) {
-        // no-op.
-      }
+      IOUtils.closeQuietly(connection);
```

Comment:
I wouldn't bother with this; we don't really do it consistently, it's not a JDK standard class, and it doesn't really save much, while adding to the dependency on commons/io

Reply:
This library should not have any non-JRE dependencies.

Reply:
I wan't aware of this requirement, reverted the change.